mgmtsystem_nonconformity/views/mgmtsystem_nonconformity.xml

```diff
@@ -210,11 +209,10 @@
     </record>
 
     <menuitem
-      id="menu_open_nonconformity"
-      action="open_mgmtsystem_nonconformity_list"
-      parent="mgmtsystem.menu_mgmtsystem_main"
-      sequence="30"
+      id="menu_open_nonconformity" 
+      action="open_mgmtsystem_nonconformity_list" 
+      parent="mgmtsystem.menu_mgmtsystem_main" 
+      sequence="20" 
```

Comment:
What is the intent beind this change? This sequence now conflicting with another menu item that's declared in a lower file.